packages/lib-modules/src/modules/utils/commands/help.js

```
@@ -6,8 +6,19 @@ async function main() {
+      // Check if the module itself is enabled
+      if (!mod.systemConfig.enabled) {
+        return [];
+      }
+
       const installedVersion = await takaro.module.moduleVersionControllerGetModuleVersion(mod.versionId);
-      return installedVersion.data.data.commands;
+      const commands = installedVersion.data.data.commands;
+
+      // Filter out disabled commands
+      return commands.filter((command) => {
+        const commandConfig = mod.systemConfig.commands && mod.systemConfig.commands[command.name];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)